pisi/atomicoperations.py

```diff
@@ -270,38 +270,39 @@ def check_operation(self):
             ipkg = self.installdb.get_package(pkg.name)
             (iversion_s, irelease_s, ibuild) = self.installdb.get_version(pkg.name)
 
-            # determine if same version
+            # determine if same distribution release
             if pkg.release == irelease_s:
                 if self.ask_reinstall:
-                    if not ctx.ui.confirm(_("Re-install same version package?")):
+                    if not ctx.ui.confirm(_("Re-install same distribution release of package?")):
                         raise Error(_("Package re-install declined"))
                 self.operation = REINSTALL
             else:
-                pkg_version = pisi.version.make_version(pkg.version)
-                iversion = pisi.version.make_version(iversion_s)
-
-                pkg_release = int(pkg.release)
-                irelease = int(irelease_s)
+                #pkg_version = pisi.version.make_version(pkg.version)
+                #ctx.ui.debug(f"New upstream version after version parsing: {pkg_version}")
+                #iversion = pisi.version.make_version(iversion_s)
+                #ctx.ui.debug(f"Installed version after version parsing: {iversion}")
+                #pkg_release = int(pkg.release)
+                #irelease = int(irelease_s)
 
                 # is this an upgrade?
                 # determine and report the kind of upgrade: version, release
-                try:
-                    if pkg_version > iversion:
-                        ctx.ui.info(_("Upgrading to new upstream version"))
-                        self.operation = UPGRADE
-                except TypeError as e:
-                    ctx.ui.error(_("Malformed package version, falling back to distribution release"))
-                    ctx.ui.debug(str(e))
+                #try:
+                #    if pkg_version > iversion:
+                #        ctx.ui.info("Upgrading to new upstream version")
+                #        self.operation = UPGRADE
+                #except TypeError as e:
+                #    ctx.ui.error(f"Comparing new upstream version '{pkg_version}' to installed version '{iversion}' failed, falling back to distribution release comparison (Add '--debug' to see full traceback)")
+                #    ctx.ui.debug(str(e))
```

Comment:
I don't like adding commented out blocks to released code. I just don't see the point- We have git history; comments just add confusion. The updated version of this code is super, super short and simple, but having this giant comment in the middle makes it look more complex than is necessary.

I advocate for outright removing code we don't need anymore. I know that hasn't always been the eopkg style, but I think we should start somewhere.